### DISCUSSION TITLE
Welcome to libre-wsdl4j Discussions!

### DISCUSSION THREAD

**Discussion by andreasrosdalw** [librewsdl4j/libre-wsdl4j#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rsearls** [comment]
Andreas I talked to the eclipse SOAP spec team.  They say no changes are expected to the WSDL api this library is based upon, so no jakarta api migration will be required.
